src/AppBundle/Entity/Repository/SurveyTakenRepository.php

```diff
@@ -22,4 +23,23 @@ public function findAllTakenBySurvey(Survey $survey):array
 
         return $qb->getQuery()->getResult();
     }
+
+    /**
+     * @param User $user
+     * @param Survey $survey
+     *
+     * @return SurveyTaken[]
+     */
+    public function findAllSurveyTakenBySurveyAndUser(Survey $survey, User $user):array
```

Comment:
`SurveyTaken` skal ikke være med i navnet for funksjoner i `SurveyTakenRepository`. Rename til `findAllBySurveyAndUser`